index.html

```diff
@@ -57,5 +57,202 @@ <h1>I’m Happi. Glad to see you!</h1>
+                    sign-ups required.
+                </p>
+            </section>
+            <ul class="webdev-languages">
+                <li> <a href="#html">HTML</a> </li>
+                <li> <a href="#css">CSS</a> </li>
+                <li> <a href="#javascript">Javascript</a></li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        </section>
+    <section class="cards">
+        <img src="./figmaImages/snapshoots/Snapshoot Portfolio-3.svg" alt="Snapshoot">
+   
+    <div class="cards-description">
+        <h2 class="cards-titles">Tonic</h2>
+        <div class="subheading-lists">
+        <span class="canopy">CANOPY</span>
+        <img src="./figmaImages/icons/circle-icon.svg" alt="">
+        <span class="subheading-list-title">Back End Dev</span>
+        <img src="./figmaImages/icons/circle-icon.svg" alt="">
+        <span class="subheading-list-title">2015</span>
+        </div>
+        <section class="card-content">
+            <p>
+                A daily selection of privately personalized reads; no accounts or 
+                sign-ups required.
+            </p>
+        </section>
+        <ul class="webdev-languages">
+            <li> <a href="#html">HTML</a> </li>
+            <li> <a href="#css">CSS</a> </li>
+            <li> <a href="#javascript">Javascript</a></li>
+        </ul>
+        <button class="see-project">See Project</button>
+    </div>
+    </section>
+<section class="cards">
+    <img src="./figmaImages/snapshoots/Snapshoot Portfolio-4.svg" alt="Snapshoot">
+
+<div class="cards-description">
+    <h2 class="cards-titles">Multi-Post
+        Stories</h2>
+    <div class="subheading-lists">
+    <span class="canopy">CANOPY</span>
+    <img src="./figmaImages/icons/circle-icon.svg" alt="">
+    <span class="subheading-list-title">Back End Dev</span>
+    <img src="./figmaImages/icons/circle-icon.svg" alt="">
+    <span class="subheading-list-title">2015</span>
+    </div>
+    <section class="card-content">
+        <p>
+            A daily selection of privately personalized reads; no accounts or 
+            sign-ups required.
+        </p>
+    </section>
+    <ul class="webdev-languages">
+        <li> <a href="#html">HTML</a> </li>
+        <li> <a href="#css">CSS</a> </li>
+        <li> <a href="#javascript">Javascript</a></li>
+    </ul>
+    <button class="see-project">See Project</button>
+</div>
+</section>
+</section>
+<section class="about-myself">
+    <h1>About Myself</h1>
+    <p class="presentation-paragraph">I’m a software developer! I can help you build a product, feature, or website. Look through
+        some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to <a href="#contact">contact me</a>
+    </p>
+    <section class="socialnetwork-part">
+        <p class="connect-link">
+            <a href="#connect">LET'S CONNECT</a>
+        </p>
```

Comment:
- [ ] As per requirement this part has to us a `hx` tag such as `h3` for semantic purposes, kindly fix this

Reply:
Okey
Thanks